py/desitarget/mock/mockmaker.py

```diff
@@ -171,6 +171,10 @@ def empty_targets_table(nobj=1):
     #targets.add_column(Column(name='NUMOBS_INIT', length=nobj, dtype='i8'))
     #targets.add_column(Column(name='HPXPIXEL', length=nobj, dtype='i8'))
 
+    # Default MASKBITS for all mock targets to non-zero.  If populating with realistic depths
+    # these will later be overwritten.
+    targets['MASKBITS'] = 2**10
+    
```

Comment:
The default `MASKBITS` should *not* be `BAILOUT` (which we're aiming to not even be needed for DR9); it should be initialized to zero with the appropriate datatype, and then replaced along with the other imaging quantities.

Reply:
Yes, sorry, you're correct obviously.  This was meant to be a fix for when quantities_at ... returned None for a failed brick, or outside the footprint, that these targets would automatically be flagged as BAILOUT.  This was @geordie666 suggestion and seemed neater than catching the None, etc cases.  Clearly I messed it up and, to retain the previous functionality, I should have had MASKBITS default to zero unless a realistic imaging mode was requested, in which case default to something non-zero unless told otherwise?